_sass/features.scss

```diff
@@ -1,239 +1,234 @@
 .features {
```

Comment:
@rahulrana701 will you please remove the unnecessary line changes or fix this lint changes. Its hard to see what is the real change.